sdk/customapp/custom_app_utils.py

```
@@ -0,0 +1,63 @@
+# Utility Class for CustomApp
+
+from typing import Any
+import argparse
+
+
+# Constants
+DEFAULT_BATCH_SIZE = 100
+DEFAULT_OUTPUT_DIR = "/tmp/customapp_export"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/customapp_export"

(S108)